edi_835_parser/segments/claim.py

```diff
@@ -21,6 +22,13 @@ def __init__(self, segment: str):
 		self.status = segment[2]
 		self.charge_amount = segment[3]
 		self.paid_amount = segment[4]
+		self.patient_responsibility_amount = get_element(segment, 5)
+		self.claim_filing_indicator_code = get_element(segment, 6)
+		self.payer_claim_control_number = get_element(segment, 7)
+		self.facility_type_code = get_element(segment, 8)
+		self.claim_frequency_code = get_element(segment, 9)
+		self.drg_code = get_element(segment, 11)
```

Comment:
i'd started to put together some unit tests for the claim segment parsing, but wanted to gauge interest before introducing a new approach